docs/UserGuide.adoc

```diff
@@ -55,6 +55,11 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 [[Features]]
 == Features
 
+===Person Panel for v1.5 'In Development'
```

Comment:
I think you need to add a space between the === and "Person Panel v1.5.." for it to display correctly.